walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/MeetingRequest.java

```diff
@@ -46,6 +46,9 @@ public MeetingRequest(Collection<String> attendees, long duration) {
    * Returns a read-only copy of the people who are optional to attend this meeting.
    */
   public Collection<String> getOptionalAttendees() {
+    if (optional_attendees == null) {
+      return Collections.unmodifiableCollection(new HashSet<>());
+    }
```

Comment:
The final variable is instantiated to empty so I think we can delete this check

Reply:
Ohh this was for when I tried to run it with the frontend, it was throwing an NPE and this fixed it